sound/soc/intel/boards/sof_sdw.c

```diff
@@ -56,6 +56,23 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 				SOF_SDW_TGL_HDMI | SOF_SDW_PCH_DMIC |
 				SOF_SSP_PORT(SOF_I2S_SSP2)),
 	},
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Intel Corporation"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "Ice Lake Client"),
+		},
+		.driver_data = (void *)(SOF_SDW_PCH_DMIC),
```

Comment:
superfluous second pair of parentheses, similar below

Reply:
changed